tools/milestone.py

```diff
@@ -0,0 +1,114 @@
+"""Script for automatically creating a milestone in repositories.
+
+Only specified repositories (via environment variable) will be handled.
+
+The purpose of this code is to connect to a specific repository
+and create a milestone. This milestone will be associated to a certain date
+provided as an input argument.
+
+"""
+
+import datetime
+import os
+
+import github
+
+# Insert your credentials... None by default
+MY_PAT = None
+
+# Provide the repository you want to create a milestone in... None by default
+REPOSITORY = None
+
+# Provide the release date to be considered... None by default
+# If you provide manual input it should be using:
+#
+#   RELEASE_DATE = datetime.datetime.strptime(date_str,"%Y/%m/%d")
+#
+# where "date_str" must be a string date of format YYYY/MM/DD
+RELEASE_DATE = None
+
+# =============================================================================
+# MODIFY WITH CAUTION FROM THIS POINT ONWARDS
+# =============================================================================
+
+# Check if a value for PAT was provided
+if MY_PAT is None:
+    # This probably means that we are creating the milestone automatically
+    # using our GitHub action: Create milestones for Ansys Release...
+    # Thus, let us read the GitHub Token.
+    print("Reading access token from 'TOKEN' environment variable...")
```

Comment:
Nitpicking:
Here the python variable is named `MY_PAT` whereas the environment variable is named `TOKEN`. It might be confusing for a user. Can we rename `MY_PAT`?

Reply:
Totally, my bad. Sorry